packages/subgraph/manifest/data/rinkeby.json

```diff
@@ -6,6 +6,16 @@
       "name": "DAORegistry",
       "address": "0xeacB900925d8300e3501df13F0Dc6930892060f4",
       "startBlock": 11168251
+    },
+    "PluginRepoRegistry": {
+      "name": "PluginRepoRegistry",
+      "address": "0x2B8C4DD137104d1E869105cd0106e7D9EF955BfE",
+      "startBlock": 7727664
+    },
+    "PluginSetupProcessor": {
+      "name": "PluginRepoRegistry",
+      "address": "0x2B8C4DD137104d1E869105cd0106e7D9EF955BfE",
+      "startBlock": 7727664
```

Comment:
when do u think we should remove rinkeby ?

Reply:
now xD